packages/cli/src/workflows/workflows.controller.ee.ts

```diff
@@ -200,20 +201,18 @@ EEWorkflowController.post(
 /**
  * (EE) GET /workflows
  */
-EEWorkflowController.get(
-	'/',
-	ResponseHelper.send(async (req: WorkflowRequest.GetAll) => {
-		const [workflows, workflowOwnerRole] = await Promise.all([
-			EEWorkflows.getMany(req.user, req.query.filter),
-			Container.get(RoleService).findWorkflowOwnerRole(),
-		]);
-
-		return workflows.map((workflow) => {
-			EEWorkflows.addOwnerId(workflow, workflowOwnerRole);
-			return workflow;
-		});
-	}),
-);
+EEWorkflowController.get('/', async (req: WorkflowRequest.GetMany, res: express.Response) => {
+	try {
+		const [workflows, count] = await EEWorkflows.getMany(req.user, req.query);
+		const role = await Container.get(RoleService).findWorkflowOwnerRole();
+		const data = workflows.map((w) => EEWorkflows.addOwnerId(w, role));
+
+		res.json({ count, data });
+	} catch (maybeError) {
+		const error = utils.toError(maybeError);
+		ResponseHelper.sendErrorResponse(res, error);
+	}
```

Comment:
This change will not stop sending these errors to Sentry. That's one of the things that `send` does.

Reply:
Added `reportError`. Decided against using `send` to add yet another param so that we can optionally have `count` at top level of the response.